src/Appwrite/Utopia/Response/Model/UsageFunctions.php

```diff
@@ -16,58 +16,59 @@ public function __construct()
                 'default' => '',
                 'example' => '30d',
             ])
-            ->addRule('executionsTotal', [
+            ->addRule('functions', [
```

Comment:
Same as my comment above. We need to maintain consistency in the way we name the metrics. we use the `Count` suffix in some places and don't use it in some..